frontend/src/components/workspaces/Dashboard/Dashboard.tsx

```diff
@@ -30,7 +29,6 @@ const Dashboard: FC<IDashboardProps> = ({ ...props }) => {
             }))}
             onClick={(id: number) => {
```

Comment:
```suggestion
            onClick={setSelectedWs}
```
It should be possible to do something alike

Reply:
Did it